mkdocs_macros/plugin.py

```diff
@@ -571,6 +570,13 @@ def render(self, markdown: str, force_rendering:bool=False):
         # expand the template
         on_error_fail = self.config['on_error_fail']
         try:
+            # If title meta variable is present,
```

Comment:
I see only processing of variables. What about macros and filters?

Reply:
This just mimics the current behavior of the existing jinja2 rendering code for the page.
https://github.com/fralau/mkdocs-macros-plugin/blob/f97565ffe66a2b50528a00f88f36c9d390c0861b/mkdocs_macros/plugin.py#L580-L583

Unless I misread the code the `on_config` method should load all the filters and macros to `self.env` and `self.variables` right? and `page_variables` in `render` is just a copy of `self.variables`. So all macros and filters should work fine like usual. 

https://github.com/fralau/mkdocs-macros-plugin/blob/f97565ffe66a2b50528a00f88f36c9d390c0861b/mkdocs_macros/plugin.py#L696-L716

Let me know if i'm wrong. I'll look into it a bit more then.